terminal_ui.py

```diff
@@ -0,0 +1,46 @@
+from game import Game
+
+
+class TerminalUI:
+    def __init__(self):
+        self.play()
+
+    def play(self):
+        game.set_up_game()
+        self.print_board()
```

Comment:
we can remove this print_board by moving the print_board in line 19 to line 16. (depends on if we want to print the board after the win or not)

Reply:
I changed it, looks much better!